doc/release-notes/release-notes-5.5.0.md

```diff
@@ -0,0 +1,479 @@
+Notable changes
+===============
+
+RPC Changes
+-----------
+
+- `getdeprecationinfo` has several changes:
+  - It now returns additional information about currently deprecated and
+    disabled features.
+  - A new `end_of_service` object that contains both the block height for
+    end-of-service and the estimated time that the end-of-service halt is
+    expected to occur. Note that this height is just an approximation and
```

Comment:
I think my comment got lost a couple of times. This should be "Note that this time..." instead of height.

Reply:
```suggestion
    expected to occur. Note that this time is just an approximation and
```

Reply:
Ahh, it was probably merged to the release notes for the RC, rather than made as a fix to the `release_notes.md` doc, which got copied out each time.